iceoryx_utils/test/moduletests/test_cxx_newtype.cpp

```diff
@@ -83,8 +83,20 @@ TEST(NewType, CopyAssignableDoesCompile)
 {
     Sut<cxx::NewType<int, newtype::ConstructByValueCopy, newtype::CopyAssignable, newtype::Comparable>> a(491), b(492),
         c(423);
+
+    // Ignore false positive
+    // See: https://isocpp.org/wiki/faq/strange-inheritance#hiding-rule
+
+#if (__GNUC__ == 9 && (__GNUC_MINOR__ > 1 || (__GNUC_MINOR__ == 1 && __GNUC_PATCHLEVEL__ > 0)))
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-copy"
     b = a;
+#pragma GCC diagnostic pop
     EXPECT_TRUE(a == b);
+#else
+    // Test disabled.
```

Comment:
why is this test disabled on everything but a specific GCC version?

Reply:
Bug in GCC apparently. Ask @elfenpiff. I would rather resolve it elsewhere though as this isn't related to thsi PR.

Reply:
I can make an issue for it.

Reply:
I think you misunderstood my question. The test is disabled for all compiler but the GCC with the bug ;)